.github/workflows/ci.yml

```diff
@@ -13,6 +13,10 @@ jobs:
         uses: actions/checkout@v2
         with:
           path: src/github.com/vmware-tanzu/astrolabe
+      - name: Setup Go version
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16.1' # The Go version to download (if necessary) and use.
```

Comment:
In K8s, 1.16.10 is used for earlier versions.  Can we use 1.16.10 instead?

Reply:
Sure